packages/component-base/src/style-props.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css } from 'lit';
+import { addGlobalThemeStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
```

Comment:
Let's also add `"@vaadin/vaadin-themable-mixin"` dependency to `package.json` of the component-base package.

Reply:
I’ll add a TODO comment.